libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -20,14 +20,15 @@
  * Code-generating part of inline assembly.
  */
 
-#include <libsolidity/inlineasm/AsmCodeGen.h>
```

Comment:
Yes, I think we should change the coding standards. I'm not sure if the order has any impact on compilation performance.